pkg/ip/ipforward.go

```diff
@@ -18,10 +18,12 @@ import (
 	"io/ioutil"
 )
 
+// EnableIP4Forward enables IPv4 forwarding
```

Comment:
Somewhere around this point, I would just filter out this particular lint warning.  There is no value in repeating the name of the function twice more.